src/ResourceManager/Websites/Commands.Websites/help/Remove-AzureRmWebApp.md

```diff
@@ -15,13 +15,13 @@ Removes an Azure Web App.
 
 ### S1
 ```
-Remove-AzureRmWebApp [-Force] [-ResourceGroupName] <String> [-Name] <String>
```

Comment:
There doesn't seem to be a test for this cmdlet.

Reply:
WebAppTests.ps1 has this "function Test-RemoveWebApp" which is updated. Is there anything else i am missing here? Thanks!